it/controllers/ContractorCYAControllerISpec.scala

```diff
@@ -39,7 +39,7 @@ class ContractorCYAControllerISpec extends IntegrationTest with ViewHelpers {
     s"/update-and-submit-income-tax-return/construction-industry-scheme-deductions/$taxYear/check-construction-industry-scheme-deductions?month=$month&contractor=$employerRef"
   }
 
-  private val anUpdateCISSubmission: CISSubmission = aCISSubmission.copy(
+  private val anUpdateCISSubmission: CISSubmission = aCreateCISSubmission.copy(
```

Comment:
the builder for the update can be used here instead of using .copy on aCreateCISSubmission